src/pages/listing/index.tsx

```diff
@@ -82,6 +88,13 @@ export default function Listing() {
     delay: 1000,
   });
 
+  useUpdate({
+    callback: () => {
+      void asyncGetContracts();
+    },
+    delay: UPDATING_INTERVAL,
+  });
+
```

Comment:
Can we update the data only if the user is in the first page?

Reply:
done